tox.ini

```
@@ -73,12 +73,12 @@ commands =
 description = Build a wheel and source distribution
 skip_install = True
 deps =
-    pep517
+    build
     twine
 commands =
     python -c "from pathlib import Path; \
                [x.unlink(missing_ok=True) for x in Path('{toxinidir}/dist').glob('*')]"
-    python -m pep517.build -s -b {toxinidir} -o {toxinidir}/dist
+    python -m build -s -w -o {toxinidir}/dist {toxinidir}
```

```suggestion
    python -m build -o {toxinidir}/dist {toxinidir}
```

This way it will build the wheel from the sdist.
https://pypa-build.readthedocs.io/en/latest/changelog.html#id10

---

Thanks!